service/logger.go

```diff
@@ -19,19 +19,24 @@ import (
 
 	"go.uber.org/zap"
 	"go.uber.org/zap/zapcore"
+
+	"github.com/open-telemetry/opentelemetry-collector/internal/version"
 )
 
 const (
-	logLevelCfg = "log-level"
+	logLevelCfg   = "log-level"
+	logProfileCfg = "log-profile"
```

Comment:
I wonder if we want to make this more generic, ie.: use `profile` for other things but this is already a good start. Let's keep as it is at least for now.